### ISSUE TITLE
False positive: empty() suggests === null when type is null|string

### ISSUE THREAD

**Issue by Azenet** [kalessil/phpinspectionsea#1183]
Hi,

| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | False-Positive                         |
| Plugin         | Php Inspections (EA Extended) 3.0.10 |
| Language level | PHP 7.3      |

Current behaviour (description/screenshot:)
--
Empty usage inspection triggers with this code :
```php
/** @var $var null|string */
$var = '';

if (empty($var)) {
    // ...
}
```

Expected behaviour
--
The inspection should not be triggered if the checked variable can be a string.

Environment details
---
PhpStorm 2018.3.3
Build #PS-183.5153.36, built on January 9, 2019
JRE: 1.8.0_152-release-1343-b26 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.20.0-arch1-1-ARCH

Thank you!

**Comment by kalessil** [comment]
Fixed!
